src/aatoolbox/utils/codab.py

```diff
@@ -0,0 +1,43 @@
+"""Retrieve COD administrative boundaries."""
+import geopandas as gpd
+
+from aatoolbox.config.aatoolbox import AaToolbox
+
+MODULE_BASE_DIR = "cod_ab"
+
+
+class CodAB(AaToolbox):
+    """
+    Work with COD AB (administrative boundaries).
+
+    Parameters
+    ----------
+    iso3 : (str)
+        country iso3
+    """
+
+    def __init__(self, iso3: str):
+        super().__init__(iso3=iso3, module_base_dir=MODULE_BASE_DIR)
+
+    def get_admin0(self):
```

Comment:
I know it was more about the config than specific codab stuff. But is there a reason that you decided to do this per admin level? Would it be an idea to instead pass the admin level as parameter? (else we need 5 functions for adm level 0 - 4)